src/Analyzers/Core/CodeFixes/GenerateConstructor/AbstractGenerateConstructorService.State.cs

```diff
@@ -405,161 +409,6 @@ private async Task<bool> TryDetermineTypeToGenerateInAsync(
             return TypeToGenerateIn?.TypeKind is (TypeKind?)TypeKind.Class or (TypeKind?)TypeKind.Struct;
         }
 
-        private void GetParameters(
```

Comment:
this helper moved to a shared static class location.  We want to be able to compute things like which existing fields/props can be initialized with the parameters we're adding in a common way across several features.